packages/opentelemetry-core/src/index.ts

```diff
@@ -29,6 +29,9 @@ export * from './correlation-context/correlation-context';
 export * from './correlation-context/propagation/HttpCorrelationContext';
 export * from './platform';
 export * from './trace/NoRecordingSpan';
+export * from './trace/sampler/AlwaysOffSampler';
```

Comment:
Could you please update the doc here: https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-core#built-in-sampler